src/core_modules/capture-core/components/FiltersForTypes/Numeric/NumericFilter.component.js

```diff
@@ -96,8 +96,7 @@ class NumericFilter extends Component<Props> implements UpdatableFilterContent<V
     }
 
     static isFilterValid(minValue?: ?string, maxValue?: ?string, type: $Values<typeof elementTypes>) {
-        if (!(NumericFilter.validateField(minValue, type).isValid &&
-            NumericFilter.validateField(minValue, type).isValid)) {
+        if (!NumericFilter.validateField(minValue, type).isValid) {
```

Comment:
```suggestion
        if (!NumericFilter.validateField(minValue, type).isValid) || !NumericFilter.validateField(maxValue, type).isValid {
```
Thanks for finding this 😊 The second test should be maxValue